src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,151 @@
+package duke;
+
+import duke.Event;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.BufferedWriter;
+import java.io.BufferedReader;
+import java.io.IOException;
+
+/**
+ * Represents a storage object that interacts with the file system.
+ */
+public class Storage {
+    protected String path;
+    public Storage(String path) {
+        this.path = path;
+    }
+
+    /**
+     * Loads the tasks from the file and returns an ArrayList of tasks.
+     *
+     * @return An ArrayList of tasks.
+     */
+    public ArrayList<Task> load() {
+        ArrayList<Task> list = new ArrayList<>();
```

Comment:
The name can be lists instead of list. 
Distinguish clearly between single-valued and multi-valued variables. (can refer Code Quality: Namings)